src/Sarif.Viewer.VisualStudio/CodeAnalysisResultManager.cs

```diff
@@ -557,5 +558,44 @@ private void RemoveTemporaryFiles()
             // Unit tests will only create one cache.
             return this.RunIndexToRunDataCache.Values.First().RemappedPathPrefixes.ToArray();
         }
+
+        // Extract selected results from original SarifLog.
+        internal SarifLog GetPartitionedLog(IEnumerable<SarifErrorListItem> listItems)
+        {
+            if (listItems.Any())
```

Comment:
>if (listItems.Any()) [](start = 12, length = 20)

invert if condition to contain less "branching"

Reply:
refactor the function